cwms-data-api/src/test/resources/cwms/cda/data/dto/location/kind/base_outlet.json

```diff
@@ -22,5 +22,6 @@
   "rating-group-id": {
     "office-id": "SPK",
     "name": "Rating-BIGH-TG1"
-  }
+  },
+  "shared-loc-alias-id": "rating spec id"
```

Comment:
This should be an example of a rating spec if

Reply:
A leftover from testing, updated to example rating spec id